### PR TITLE
Feature/setup calendar

### PR DESCRIPTION
**Pull request by kapil-31**
Screenshot 2024-10-22 at 10 55 07 am

### CODE REVIEW

**Review by techgaun — CHANGES_REQUESTED** [review]
build folder should not be committed ideally. can you figure out how we can achieve that so that the source code is more cleaner?

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+npm install --save-dev --save-exact prettier
```
Comment (techgaun) [inline .prettierignore]: what does this refer to? shouldn't this be ignoring files instead?

File: src/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (techgaun) [inline src/styles/main.css:3]: I'm thinking tailwind thik chha but should be an optional dependency and we should provide a way to let users choose their own library/solution for styling

Comment (techgaun) [reply src/styles/main.css:3]: this could be a follow up worker

**Review by techgaun — CHANGES_REQUESTED** [review]

File: src/styles/main.css (hunk 2 above)
Comment (kapil-31) [reply src/styles/main.css:3]: this only for package itself we'dont force user to use tailwind user can use whatever css  they want

File: .prettierignore (hunk 1 above)
Comment (kapil-31) [reply .prettierignore]: removed

**Review by techgaun — APPROVED** [review]

**Pull request merged**